pkgs/games/brogue/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, SDL, ncurses, libtcod, binutils }:
+
+stdenv.mkDerivation rec {
+  name = "brogue-${version}";
+  version = "1.7.4";
+
+  src = fetchurl {
+    url = "https://sites.google.com/site/broguegame/brogue-${version}-linux-amd64.tbz2";
+    sha256 = "1lygf17hm7wqlp0jppaz8dn9a9ksmxz12vw7jyfavvqpwdgz79gb";
+  };
+
+  prePatch = ''
+    sed -i Makefile -e 's,LIBTCODDIR=.*,LIBTCODDIR=${libtcod},g' \
+                    -e 's,sdl-config,${SDL}/bin/sdl-config,g'
+    sed -i src/platform/tcod-platform.c -e "s,fonts/font,$out/bin/fonts/font,g"
+    make clean
+    rm -rf src/libtcod*
+  '';
+
+  buildInputs = [ SDL ncurses binutils libtcod ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r bin/* $out/bin/
```

Comment:
This will install a lot of crud into $PATH which will show up during command search, because the png files etc will have the executable bit set, so this is no good.